modules/articles/server/policies/articles.server.policy.js

```diff
@@ -20,6 +20,9 @@ exports.invokeRolesPolicies = function () {
     }, {
       resources: '/api/articles/:articleId',
       permissions: '*'
+    }, {
+      resources: '/api/parameterized-query/articles',
```

Comment:
I see why you called this route parameterized-query, and we probably want to keep backwards-compatibility, but it does make me wonder whether we could retitle the existing articles routes /api/articles/list and /api/articles/list/:articleId and rename your query /api/articles/query